src/mercury-inventory/mercury/inventory/server.py

```diff
@@ -18,39 +18,40 @@
 import zmq.asyncio
 
 from mercury.common.asyncio import dispatcher, transport
-from mercury.common.log import log_format
 from mercury.inventory.controller import InventoryController
 from mercury.inventory.configuration import get_inventory_configuration
 
 log = logging.getLogger(__name__)
 
 
 class InventoryServer(transport.AsyncRouterReqService):
-    def __init__(self, bind_address, inventory_configuration):
+    def __init__(self, bind_address, config):
         super(InventoryServer, self).__init__(bind_address)
 
-        inventory_controller = InventoryController(inventory_configuration)
+        inventory_controller = InventoryController(
+            config.inventory.db.servers,
+            config.inventory.db.replica_name,
+            config.inventory.db.name,
+            config.inventory.db.collection)
+
         self.dispatcher = dispatcher.AsyncDispatcher(inventory_controller)
 
     async def process(self, message):
         return await self.dispatcher.dispatch(message)
 
 
-def main(bind_address='tcp://0.0.0.0:9000',
-         log_level='INFO',
-         asyncio_debug=False,
-         configuration_file=None):
-    logging.basicConfig(level=logging.getLevelName(log_level),
-                        format=log_format)
+def main():
+    config = get_inventory_configuration()
+
+    logging.basicConfig(level=logging.getLevelName(config.log_level),
+                        format=config.log_format)
```

Comment:
if would be _nice_ to use `logging.config.dictConfig`...

Reply:
👍

Reply:
bare minimum logging is implemented right now.. There will be another PR to address logging configuration overall.